### DISCUSSION TITLE
posts/2024/09/my-vision-of-yefu-zheng/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#68] [General]
# posts/2024/09/my-vision-of-yefu-zheng/

https://guozheng.rbind.io/posts/2024/09/my-vision-of-yefu-zheng/

**Comment by XiangyunHuang** [comment]
前段时间，我看卢晖临的主页，顺便翻了翻其他人的，其中就有郑也夫的，【不接受/申请任何官方的钱】。我想到研究社会学需要的客观性，我也想到卢教授那篇革命前后中国乡村变迁的文章中的一个脚注（目的是说明立场，我没有被收买，我做的东西是相对客观的）

关于费孝通，鼎鼎大名，一直被称为社会学的奠基人。我初次听到是从高华教授的课程《重读中华人民共和国历史》。讲建国初知识分子的思想改造运动，费孝通等带头做检讨和思想改造。晚年，费老对此事表示对不起知识分子。我对费老的著作一本也没看，社会学东西很多，本着获取常识性知识和结合自己的问题出发的目的，我还不想深入和拓宽。P.S. 萧公权那本《中国乡村》还有一部分没看完。

我也注意到一个现象，就是各大书店的社会学书架上，大体上有两类书。一类是翻译自国外的，讲的概念和问题可能同样适用于中国。另一类是上个世纪费孝通、潘光旦等人的专著、文集。似乎很少有中国改革开放以后的，当然，从网络上查找，可以找到一些。放眼国内，整体上现有的书籍和讯息比较旧，当下的问题可能比较新，至少含有新东西，似乎只有关注国内有关社会学大佬的主页，看他们发表的论文，才能跟进最新进展。
还有，我注意到你看的书似乎绝大部分来自国外的，但是，你研究的问题、做的田野调查是中国本土的，我感觉这其中有沟壑，需要你去填补呀。你怎么看这个问题？你怎么填补这个沟壑？

**Reply by XiangyunHuang** [reply]
我突然发现自己在自问自答，我的疑惑可以从这些社会学家的主页中解答出来，正如看卢教授的文章解答我对中国乡村的一些困惑。填补这所谓的沟壑也是，可以不要管书店的售卖，那是选择性展示，那是意识形态管控和金钱指标约束下的筛选，学者主页上挂的书籍和文章，属于学术范围，学术领域的管控相对宽松一些，可以看到更多东西。

**Reply by residualsun1** [reply]
> 费孝通等带头做检讨和思想改造。晚年，费老对此事表示对不起知识分子。

我觉得在当时也是身不由己。在我看来，后人对于前人境况的认识往往只有流传下来的文字和话语，多是后见之明，是没有权力去指摘前人的。而且费老还是挺不容易的，我不大喜欢别人去神化他，但他的贡献不能被忽视。

> 我也注意到一个现象，就是各大书店的社会学书架上，大体上有两类书。一类是翻译自国外的，讲的概念和问题可能同样适用于中国。另一类是上个世纪费孝通、潘光旦等人的专著、文集。似乎很少有中国改革开放以后的，当然，从网络上查找，可以找到一些。放眼国内，整体上现有的书籍和讯息比较旧，当下的问题可能比较新，至少含有新东西，似乎只有关注国内有关社会学大佬的主页，看他们发表的论文，才能跟进最新进展。

50年代以后，社会学与人类学学科被「打倒」的那段历史其实影响很大，改革开放恢复学科以后，老一辈的社会学家虽然紧锣密鼓地开展重建工作，但是国内国外产生的差距还是太大了，而且社会学本身就起源于国外，我们得先知道别人的理论发展到什么地步，然后再去谈「本土化」。但是这中间还是有不少曲折，我自己的感觉是国内意识形态的生态还是对此有很多影响，然后一些掌握资源的大佬们并没有开拓、紧跟国际的意愿（关于这一点，值得一提的是浙大的赵鼎新老师，他辞去芝加哥大学的社会学终身教职，来到浙大支持社会学系，引进许多国际的学者，并希望能让他们不要面对太多发表压力，沉浸研究，这个愿景是美好的，但终究还是遭遇了行政等层面的阻力），种种原因，可以说我们整体和国际应当还有断层，可见的书籍是要赶紧「补课」的，尚且没有能力引进和翻译出许多前沿。不过我觉得现在可能也在改善了，还是有一些比较新的书冒出来的，只是可能还不成规模，不是很明显。说到这，我觉得你可以稍微了解一下华东师范大学做的「薄荷实验」系列书籍，这个系列就致力于比较晚近的社会学和人类学书籍（偏人类学多一些），是近年来很受欢迎的系列。然后，我听说其实出版业在中国的境况并不算好，似乎有些声音是「中国的出版业已死」，但我不知道为什么。不过相对比较清楚的是，中国的学术翻译生态可能不好，至少相对台湾，中国的社科书籍翻译可没少被诟病。中国的翻译吃力不讨好，不如写文章投文章，有些书籍的翻译还是大佬召集一批人，你翻一点，我翻一点，然后出版。

前沿内容确实在大佬论文那儿多见一些，或者去参加一些学术会议。

> 还有，我注意到你看的书似乎绝大部分来自国外的，但是，你研究的问题、做的田野调查是中国本土的，我感觉这其中有沟壑，需要你去填补呀。你怎么看这个问题？你怎么填补这个沟壑？

我觉得这是我要思考的问题，但从某种程度上来说，这还不是当下这个阶段的我要真正解决的问题。我觉得我本身对这个问题的理解也是模糊的，我不知道我应该成为一个怎样的社会学学生，我觉得这个问题在情理上是应当解答的，但我又觉得还没有能力解答。具体来说，对于许多本科生的学习而言，其实能够充分把握一个理论的内容，讲清楚该理论就挺厉害了（像社会学三大家，韦伯、涂尔干、马克思他们的理论至今还被拿来进行研究和阐释）。我们说写论文往往要有一个创新点，但是，有多少学生能很自信地说这是一个真正的「创新」，是前人没有的？以我的田野调查为例，我目前的计划是研究硇洲岛上的神明轮祭仪式，参考日本学者提出，并由台湾地区人类学家发展出来的「祭祀圈」理论，但对于我来说，其实很多时候我是在「套用」理论，在「尽力」地做到能用理论解释当地的现象，先做到这一点是当前的目标，而后续才会有许多反思出现。因为我现在对理论和实际都还是陌生的感觉，我想我要先借助一个「脚手架」进入，然后才能慢慢知道沟壑是什么，怎么填补。据我个人的经验，往往是在一个学者有了一些积累以后，才会走向反思、提出质疑，然后修正，甚至提出自己的理论，这时候才是填补沟壑的时候，而这往往是层次比较高的学者做的。比如我刚刚说到的「祭祀圈」理论，它是日本学者冈田谦为了解释台湾地区根据神明信仰划分出的人群界限提出的概念，后来庄英章、许嘉明、施振明等学者又在这个概念上不断细化，林美容还提出了「信仰圈」，多年以后，张珣又提出要「打破圈圈」，觉得过去的研究没有把握好台湾地方社会的形成——我觉得这就是一个填补沟壑的过程。当然，这是从一个理论的角度来看，从更大的角度来看，还有施坚雅的「市场理论」、弗里德曼的「中国宗族理论」，这些理论和「祭祀圈」般，都是为了解释中国地方社会如何形成，它们各自有所道理，被人支持，也被人质疑，并且在质疑的过程中不断发展，是否「本土」也是它们被不断讨论和发展的重要原因。学者们可以在这个沟壑里做出许多讨论，可以说应当没有尽头，而这可能也是理解复杂社会的常态，毕竟社会科学给我们的只有各种视角，没有标准的答案。

但我觉得你的问题意识反而比我敏锐，因为我会下意识地忽略所谓的「沟壑」——尽管我知道我现在可能还没有能力解答，但我理应循着这个方向去思考。

**Comment by XiangyunHuang** [comment]
哈哈，有时候，我提的问题可能对你产生了挑战，但是，我总能在你的回答中获得一些收获。我把问题抛出来的时候，并没有考虑你是本科生还是博士生，你是双非还是国际名校的学生，因为，我觉得没必要考虑。一切从问题出发，包括你本科做毕业论文、做田野调查也是，我发现社会学的田野调查是一切的源头。这个源头在经过不同人的观察和调查后，可能因时因地因人有不同的观点和材料产生。历史已有的研究很多，从自己研究的对象出发，沉下去才可能发现有价值的新问题，不然，只能写点浮于表面的综述。


关于沉下去的问题，我不知道你做田野调查的时候，和当地人融入、深入到什么程度，聊到什么程度，能不能感同身受。比如，这个祭祀对他们的意义是什么？（家庭，家族，村落层面）他们对这个祭祀的态度的变迁，以及变迁的原因是什么？（经济、文化、行政）。可能我正好在社会学外面，反而没有社会学知识的圈圈围绕，可以尽情提问，也许幼稚，也许又有思维火花。


我觉得我还得了一个便宜，就是我是一个刚刚接触社会学的外行人，所以，我提的问题即使很幼稚也一点也不觉得羞耻了。


Xiangyun,

Best regards.

在 2024年11月17日，01:10，Guozheng Huang ***@***.***> 写道：

﻿

费孝通等带头做检讨和思想改造。晚年，费老对此事表示对不起知识分子。

我觉得在当时也是身不由己。在我看来，后人对于前人境况的认识往往只有流传下来的文字和话语，多是后见之明，是没有权力去指摘前人的。而且费老还是挺不容易的，我不大喜欢别人去神化他，但他的贡献不能被忽视。

50年代以后，社会学与人类学学科被打倒的那段历史其实影响很大，改革开放恢复学科以后，老一辈的社会学家虽然紧锣密鼓地开展重建工作，但是国内国外产生的差距还是太大了，而且社会学本身就起源于国外，我们得先知道别人的理论发展到什么地步，然后再去谈「本土化」。但是这中间还是有不少曲折，我自己的感觉是国内意识形态的生态还是对此有很多影响，然后一些掌握资源的大佬们并没有开拓、紧跟国际的意愿（关于这一点，值得一提的是浙大的赵鼎新老师，他辞去芝加哥大学的社会学终身教职，来到浙大支持社会学系，引进许多国际的学者，并希望能让他们不要面对太多发表压力，沉浸研究，这个愿景是美好的，但终究还是遭遇了行政等层面的阻力）关于你说的书架现象，我书店去得比较少，但觉得现在可能也在改善书籍和讯息比较旧的情况，还是有一些比较新的书冒出来的，但可能还不成规模，不是很明显。说到这，我觉得你可以稍微了解一下华东师范大学做的「薄荷实验」系列书籍，这个系列就致力于比较晚近的社会学和人类学书籍（偏人类学多一些），是近年来很受欢迎的系列。然后，我听说其实出版业在中国的境况并不算好，似乎有些声音是「中国的出版业已死」，但我不知道为什么。不过相对比较清楚的是，中国的学术翻译生态可能不好，至少相对台湾，中国的社科书籍翻译可没少被诟病。中国的翻译吃力不讨好，不如写文章投文章，有些书籍的翻译还是大佬召集一批人，你翻一点，我翻一点，然后出版。

前沿内容确实在大佬论文那儿多见一些，或者去参加一些学术会议。

还有，我注意到你看的书似乎绝大部分来自国外的，但是，你研究的问题、做的田野调查是中国本土的，我感觉这其中有沟壑，需要你去填补呀。你怎么看这个问题？你怎么填补这个沟壑？

我觉得这是我要思考的问题，但从某种程度上来说，这还不是当下这个阶段的我要真正解决的问题。我觉得我本身对这个问题的理解也是模糊的，我不知道我应该成为一个怎样的社会学学生，我觉得这个问题在情理上是应当解答的，但我又觉得还没有能力解答。具体来说，对于许多本科生的学习而言，其实能够充分把握一个理论的内容，讲清楚该理论就挺厉害了（像社会学三大家，韦伯、涂尔干、马克思他们的理论至今还被拿来进行研究和阐释）。我们说写论文往往要有一个创新点，但是，有多少学生能很自信地说这是一个真正的「创新」，是前人没有的？以我的田野调查为例，我目前的计划是研究硇洲岛上的神明轮祭仪式，参考日本学者提出，并由台湾地区人类学家发展出来的「祭祀圈」理论，但对于我来说，其实很多时候我是在「套用」理论，在「尽力」地做到能用理论解释当地的现象，先做到这一点是当前的目标，而后续才会有许多反思出现。因为我现在对理论和实际都还是陌生的感觉，我想我要先借助一个「脚手架」进入，然后才能慢慢知道沟壑是什么，怎么填补。据我个人的经验，往往是在一个学者有了一些积累以后，才会走向反思、提出质疑，然后修正，甚至提出自己的理论，这时候才是填补沟壑的时候，而这往往是层次比较高的学者做的。比如我刚刚说到的「祭祀圈」理论，它是日本学者冈田谦为了解释台湾地区根据神明信仰划分出的人群界限提出的概念，后来庄英章、许嘉明、施振明等学者又在这个概念上不断细化，林美容还提出了「信仰圈」，多年以后，张珣又提出要「打破圈圈」，觉得过去的研究没有把握好台湾地方社会的形成——我觉得这就是一个填补沟壑的过程。当然，这是从一个理论的角度来看，从更大的角度来看，还有施坚雅的「市场理论」、弗里德曼的「中国宗族理论」，这些理论和「祭祀圈」般，都是为了解释中国地方社会如何形成，它们各自有所道理，被人支持，也被人质疑，并且在质疑的过程中不断发展，是否「本土」也是它们被不断讨论和发展的重要原因。学者们可以在这个沟壑里做出许多讨论，可以说应当没有尽头，而这可能也是理解复杂社会的常态，毕竟社会科学给我们的只有各种视角，没有标准的答案。

但我觉得你的问题意识反而比我敏锐，因为我会下意识地忽略所谓的「沟壑」——尽管我知道我现在可能还没有能力解答，但我理应循着这个方向去思考。

**Reply by residualsun1** [reply]
> 哈哈，有时候，我提的问题可能对你产生了挑战

> 我觉得我还得了一个便宜，就是我是一个刚刚接触社会学的外行人，所以，我提的问题即使很幼稚也一点也不觉得羞耻了。

坦白说，有时候确实是会有挑战性的哈哈，但从我个人角度来看，从不觉得是幼稚，反而是觉得能有这样的问题交流而感到高兴和难得 : D。

> 关于沉下去的问题，我不知道你做田野调查的时候，和当地人融入、深入到什么程度，聊到什么程度，能不能感同身受。比如，这个祭祀对他们的意义是什么？（家庭，家族，村落层面）他们对这个祭祀的态度的变迁，以及变迁的原因是什么？（经济、文化、行政）。可能我正好在社会学外面，反而没有社会学知识的圈圈围绕，可以尽情提问，也许幼稚，也许又有思维火花。

下个学期就要开始着手写作论文了，你提到的这些问题也都很值得我去思考。我还会找机会回硇洲岛的。

**Comment by scbg-wangshu** [comment]
作者你好！在Google搜索郑也夫从而读到这篇文章，十分敬佩你在碎片化时代仍然坚持写长文来输出自己的思辨观点。祝你研究生生涯顺利，期待更多在地的研究产出！另，考虑将黄底的注释部分统一列在正文后吗？我在最初读到被注释断掉的语句时倍感困惑哈哈，供你参考。

**Reply by residualsun1** [reply]
> 作者你好！在Google搜索郑也夫从而读到这篇文章，十分敬佩你在碎片化时代仍然坚持写长文来输出自己的思辨观点。祝你研究生生涯顺利，期待更多在地的研究产出！

朋友你好！欢迎光临小站，也非常非常感谢你的阅读留言和肯定╰(*°▽°*)╯！!!（这对我来说是很高的评价

> 另，考虑将黄底的注释部分统一列在正文后吗？我在最初读到被注释断掉的语句时倍感困惑哈哈，供你参考。

没问题的！PC 端阅读可以保留脚注右浮动，但是手机端上阅读会使得阅读体验像是被「断掉」了，很奇怪。感谢你的建议！我会去调整响应式的代码的！
